rs/rust_canisters/call_loop_canister/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary", "rust_test")
+load("//bazel:canisters.bzl", "rust_canister")
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
Pre-approving, but adding comment to be resolved after a CODEOWNERS file change is made so nns team isn't the owner.